doc/plotting.rst

```diff
@@ -213,16 +213,69 @@ plot, use `plot_type='nichols'`::
 .. image:: freqplot-siso_nichols-default.png
 
 Another response function that can be used to generate Bode plots is
-the :func:`~ct.gangof4` function, which computes the four primary
+the :func:`~control.gangof4` function, which computes the four primary
 sensitivity functions for a feedback control system in standard form::
 
-    proc = ct.tf([1], [1, 1, 1], name="process")
-    ctrl = ct.tf([100], [1, 5], name="control")
-    response = rect.gangof4_response(proc, ctrl)
-    ct.bode_plot(response)	# or response.plot()
+  proc = ct.tf([1], [1, 1, 1], name="process")
+  ctrl = ct.tf([100], [1, 5], name="control")
+  response = rect.gangof4_response(proc, ctrl)
+  ct.bode_plot(response)	# or response.plot()
 
 .. image:: freqplot-gangof4.png
 
+Nyquist analysys can be done using the :func:`~control.nyquist_response`
+function, which evaluates an LTI system along the Nyquist contour, and
+the :func:`~control.nyquist_plot` function, which generates a Nyquist plot::
+
+  sys = ct.tf([1, 0.2], [1, 1, 3, 1, 1], name='sys')
+  nyquist_plot(sys)
+
+.. image:: freqplot-nyquist-default.png
+
+The :func:`~control.nyquist_response` function can be used to compute
+the number of encirclement of the -1 point and can return the Nyquist
+contour that was used to generate the Nyquist curve.
+
+By default, the Nyquist response will generate small semicircles around
+poles that are on the imaginary axis.  In addition, portions of the Nyquist
+curve that far from the origin are scaled to a maximum value, with the line
```

Comment:
```suggestion
curve that are far from the origin are scaled to a maximum value, while the line
```
The above is my minimal proposed change, but it might be better to split this into 2 sentences.